student.rb

```diff
@@ -8,4 +8,13 @@ def initialize(classroom, age, name = 'unknown', parent_permission: true)
   def play_hookey
     "¯\(ツ)/¯"
   end
+
+  def add_classroom(classroom)
+    @classroom = classroom
+    classroom.add_student(self)
```

Comment:
- [ ] What if the classroom already includes that student? You can have duplicate values. You didn't handle that case. Let's fix that